src/sites_toolbox.py

```diff
@@ -308,22 +317,142 @@ def sites_toolbox_page():
     )
     with st.expander("Input new site data"):
         with connection.get_session() as session:
-            client_site_id = st.text_input("Enter client_site_id")
-            client_site_name = st.text_input("Enter client_site_name")
-            latitude = st.text_input("Enter latitude")
-            longitude = st.text_input("Enter longitude")
-            capacity_kw = st.text_input("Enter capacity_kw")
-            created_utc = datetime.now(timezone.utc).strftime("%Y-%m-%d %H:%M:%S")
-            if st.button("Create site"):
-                site, message = create_new_site(
-                    session=session,
-                    ml_id=len(site_uuid_list) + 1,
-                    client_site_id=client_site_id,
-                    client_site_name=client_site_name,
-                    latitude=latitude,
-                    longitude=longitude,
-                    capacity_kw=capacity_kw,
-                    created_utc=created_utc,
-                )
-                st.write(message)
-                st.wrtie("New site details:", site)
+            max_ml_id = session.query(func.max(SiteSQL.ml_id)).scalar()
+            if max_ml_id is None:
```

Comment:
becasue this file is getting very long, is it worth breaking it up a bit like
- new directory called sites_toolbox
- file called create_site
- file called read_site_and_user
- file called update_site_group
or something like that? Maybe that can be in a seperate PR

Reply:
great idea.